spec/spec_helper.rb

```diff
@@ -17,8 +17,12 @@
 RSpec.configure do |config|
   require 'codeclimate-test-reporter'
   CodeClimate::TestReporter.start
+
   require 'coveralls'
   Coveralls.wear!
+  require 'webmock/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.